jenkins/stretchtests.jpl

```diff
@@ -4,12 +4,14 @@ import org.kernelci.debian.RootFS
 def r = new RootFS()
 
 def config = ['name':"stretchtests",
-              'arch_list':["armhf", "armel", "arm64", "i386", "amd64",
-                           "mips", "mipsel"],
+//              'arch_list':["armhf", "armel", "arm64", "i386", "amd64",
+//                           "mips", "mipsel"],
```

Comment:
Did you mean to leave these commented lines here

Reply:
Of course not :)  Look at the commit history, this is to quickly test the job with only one arch.